ckanext/qdes_schema/templates/scheming/package/snippets/package_form.html

```diff
@@ -56,8 +62,22 @@ <h3>{{ field.sub_heading|capitalize }}</h3>
   {%- if 'resource_fields' not in schema -%}
     <!-- force controller to skip resource-editing step for this type -->
     <input type="hidden" name="_ckan_phase" value="" />
+    {# Javscript to remove `_ckan_phase` inputs... #}
+    {% asset 'qdes_schema/dataservice' %}
   {%- endif -%}
 {% endblock %}
 
 {% block metadata_fields %}
 {% endblock %}
+
+{% block save_button_text %}
+  {% if dataset_type == 'dataservice' %}
+    {% if form_style == 'new' %}
```

Comment:
@MarkCalvert changed this comparison to `new` because it was showing "Update Data Service" on new form:
![image](https://user-images.githubusercontent.com/26393708/97488558-44e35b80-19c3-11eb-9fee-e2b8b430e8fe.png)